### PR TITLE
Release branch sprint 02

### PR DESCRIPTION
**Pull request by felipecembranelli**
please review my changes releated to sprint-02

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,5 +1,9 @@
 <html>
+    <titlE>poc 2</titlE>
```
Comment (felipecembranellirev) [inline index.html]: improve the title description and use capital

**Review by felipecembranellirev — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (felipecembranelli) [reply index.html]: I am following the convention

Comment (felipecembranellirev) [reply index.html]: please do it

Comment (felipecembranelli) [reply index.html]: done

**Review by felipecembranellirev — APPROVED** [review]

**Review by felipecembranellirev — CHANGES_REQUESTED** [review]
empty file

**Review by felipecembranellirev — APPROVED** [review]

**Pull request merged**